src/js/utils/html-utils.js

```diff
@@ -86,17 +86,17 @@
          * HtmlUtils.interpolateHtml(
          *     'You are enrolling in {spanStart}{courseName}{spanEnd}',
          *     {
-         *         courseName: 'Rock & Roll 101',
+         *         courseName: '<b>Rock</b> & Roll 101',
```

Comment:
I don't think adding the `<b>` really helps here, because it would look like a bug in the final HTML.

Reply:
Do you have a suggestion for an example showing html actually being escaped?

Reply:
I was going to say the `&` is escaped.  Maybe that is why you resolved this?

Reply:
I had thought of another example, but yep what you suggested is good. :+1: